lib/utils/dateTestUtils.ts

```
@@ -0,0 +1,326 @@
+  console.log("- validateManufacturingDate(date) - Validate a specific date");
+  console.log("- formatDateForInput(date) - Format date for HTML input");
+  console.log("- getDateValidationInfo(date) - Get detailed validation info");
+
+  // Run tests automatically
+  console.log("\n📋 Running tests...");
+  runDateTests();
+}
+
+// Make debug function available globally for browser console access
+if (typeof window !== "undefined") {
+  (window as any).debugManufacturingDate = debugManufacturingDate;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)